viewing_party/main.py

```diff
@@ -0,0 +1,237 @@
+
+def create_movie(title, genre, rating):
+    if title == None:
+        return None
+    elif genre == None:
+        return None
+    elif rating == None: 
+        return None
+    else:
+        movie_dict = {
+            'title': title, 
+            'genre': genre, 
+            'rating': rating
+        } 
+    print("movie dictionary:",movie_dict)
```

Comment:
As you noted, its usually best to remove "debugging" print statements like this before submitting.